README.md

```diff
@@ -43,6 +43,23 @@ Then deploy the application:
 helm upgrade --install -n $NAMESPACE trustify charts/trustify --values values-minikube.yaml --set-string appDomain=$APP_DOMAIN
 ```
 
+> With traces enabled:
+
+
+```bash
+helm upgrade --install --dependency-update -n $NAMESPACE infrastructure charts/trustify-infrastructure --values values-minikube.yaml --set-string keycloak.ingress.hostname=sso$APP_DOMAIN --set-string appDomain=$APP_DOMAIN --set jaeger.enabled=true --set-string jaeger.allInOne.ingress.hosts[0]=jaeger$APP_DOMAIN --set tracing.enabled=true --timeout 10m
+```
+
+Get the OTEL collector service name:
+
+```bash
+kubectl get svc
+```
+
+```bash
+helm upgrade --install -n $NAMESPACE trustify charts/trustify --values values-minikube.yaml --set-string appDomain=$APP_DOMAIN --set tracing.enabled=true --set-string collector="http://infrastructure-otelcol:4317"
```

Comment:
Could we put a default value into the `values.yaml` of the different installations (minikube, kind) and only override when necessary?

Reply:
:man_shrugging:  

* I suspect `"127.0.0.1"` | `"0.0.0.0"` | `"localhost"` none of those will work
* I'll see if we can have something like `{{ foobar }}-otelcol:4317` ..

Reply:
The default value was added in: 
`charts/trustify/templates/helpers/_infrastructure.tpl`

I have no idea how to make that in `values.yaml` after a good amount of attempts :+1: